spring-data-jdbc/src/main/java/org/springframework/data/jdbc/core/convert/JdbcIdentifierBuilder.java

```diff
@@ -41,30 +43,17 @@ public static JdbcIdentifierBuilder empty() {
 	/**
 	 * Creates ParentKeys with backreference for the given path and value of the parents id.
 	 */
-	public static JdbcIdentifierBuilder forBackReferences(PersistentPropertyPath<RelationalPersistentProperty> path,
-			@Nullable Object value) {
+	public static JdbcIdentifierBuilder forBackReferences(PersistentPropertyPathExtension path, @Nullable Object value) {
```

Comment:
We probably should keep this API id-column-count agnostic. `JdbcIdentifierBuilder` wasn't introduced with DATAJDBC-359 but that's a potential candidate for likely breaking changes as soon as we start supporting composite keys (Id-class or flat).